cmd/run/run.go

```diff
@@ -720,6 +739,9 @@ func (s *ServerContext) Run(ctx context.Context, config *Config) error {
 		server.WithListObjectsMaxResults(config.ListObjectsMaxResults),
```

Comment:
Instead of log.Warn, I think we just log that check query cache is enabled (s.Logger.Info)  Warn implies something is wrong.  Turning on query cache is an explicit configuration.